docs/source/rqaoa.rst

```diff
@@ -14,14 +14,14 @@ The development of this method is associated with an internal research project a
 
 Custom RQAOA
 ------------
-.. autoclass:: openqaoa.rqaoa.custom_rqaoa
+.. autoclass:: openqaoa.workflows.optimizer.RQAOA
```

Comment:
Need to review whether it makes sense to do the distinction now. Its the same object now

Reply:
Yes, I think it doesn't make sense to have the distinction now

Reply:
And I don't know if it should be here or only in workflows.rst

Reply:
true that! Probably not since this function is not present in this file anymore

Reply:
Since the PR originates from outside OQ I don't think I can build the docs ... but they will be automatically built once we merge the PR to dev.

Then we can fix the documentation (if needs be)